src/lab6.py

```diff
@@ -274,7 +266,10 @@ def parse(self):
         while self.i < len(self.s):
             try:
                 selector = self.selector()
+                self.literal("{")
```

Comment:
Was this a bug previously?

Reply:
No, this is kind of subtle. Before, the `block` production included the curly braces, now it's in the overall `parse` construction. `block` now just has the property/value pairs. I want to see if I can use that to parse style attributes and thereby shorten the chapter a bit and also improve the code.